lib/from_honeybee/simulation/parameter.rb

```diff
@@ -200,6 +200,18 @@ def create_openstudio_objects
             end
           end
         end
+        if @hash[:output][:summary_reports]
+          @hash[:output][:summary_reports].each do |report|
+            begin
+              os_report = OpenStudio::Model::OutputTableSummaryReports.new(@openstudio_model)
+            rescue NameError
```

Comment:
Shouldn't this be outside of the "for" loop? It seems like the SDK is set up to just create one `OutputTableSummaryReports` object  to which you can add as many summary reports as you want.

Reply:
Yes, I was interpreting it incorrectly